staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -66,6 +67,9 @@ type AuditLogOptions struct {
 	MaxBackups int
 	MaxSize    int
 	Format     string
+	// Preferred group version for the log output.
+	// Defaults to audit.k8s.io/v1beta1.
+	GroupVersionString string
```

Comment:
not even preferred, right? This version is the one and only version.

Reply:
Yup.